hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -381,7 +381,15 @@ public static HoodieTableMetaClient initTableAndGetMetaClient(Configuration hado
     return metaClient;
   }
 
-  // HELPER METHODS TO CREATE META FILE NAMES
+  /**
+   * Helper method to scan all hoodie-instant metafiles.
+   *
+   * @param fs Fs implementation for this table
+   * @param metaPath MetaPath where meta files are stored
+   * @param nameFilter NameFilter to filter meta files
```

Comment:
I'd suggest changing `Fs`, `MetaPath` and `NameFilter` to `The file system`, `The meta path` and `The name filter`

Reply:
@yanghua，Thanks for your suggestion, I'll fixed it.